manifest.yml

```diff
@@ -0,0 +1,74 @@
+app-id: com.ilnonop.sonnewebapp
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
```

Comment:
24.08 is the current

Reply:
it was indeed a request for you to use the latest runtime. And see below about how tkinter is in that version.